### PR TITLE
app-misc/neofetch: bump to 2.0

### PR DESCRIPTION
**Pull request by konimex**
Package-Manager: portage-2.3.0

On an related note, can I make a stabilization request for this package? Is there any procedure for it?

### CODE REVIEW

**Comment by mgorny** [comment]
> On an related note, can I make a stabilization request for this package? Is there any procedure for it?

Yes, you can. https://devmanual.gentoo.org/keywording/index.html

I'd say the procedure will be a bit different though — file a stablereq bug (but don't CC arches), state which version should be stabilized (and anything possibly relevant to arch teams), and wait for proxy-maint member to confirm it.

File: app-misc/neofetch/neofetch-2.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+DESCRIPTION="Simple information system script"
+HOMEPAGE="https://github.com/dylanaraps/neofetch"
+SRC_URI="https://github.com/dylanaraps/${PN}/archive/${PV}/${P}.tar.gz"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="X"
+
+DEPEND="
+	app-shells/bash:*
```
Comment (mgorny) [inline app-misc/neofetch/neofetch-2.0.ebuild]: This is most likely incorrect. Slot `:0` installs `bash`, other slots install `bash-X.Y` binaries which your program will likely not use.

File: app-misc/neofetch/neofetch-2.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	sys-apps/pciutils"
```
Comment (mgorny) [inline app-misc/neofetch/neofetch-2.0.ebuild]: Why does it need pciutils at build time? This looks a bit suspicious to me.

File: app-misc/neofetch/neofetch-2.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	X? (
+		x11-apps/xprop
+		x11-apps/xwininfo
+		x11-apps/xrandr
+		www-client/w3m[imlib]
+		media-libs/imlib2
+		media-gfx/imagemagick
+	)"
+
+	pkg_postinst() {
```
Comment (mgorny) [inline app-misc/neofetch/neofetch-2.0.ebuild]: Please don't indent function name and closing brace.

File: app-misc/neofetch/neofetch-2.0.ebuild (hunk 2 above)
Comment (mgorny) [inline app-misc/neofetch/neofetch-2.0.ebuild]: Nah, just the newest. But check it, don't follow my guesses bluntly ;-P.

Comment (konimex) [inline app-misc/neofetch/neofetch-2.0.ebuild]: I went ahead and deleted bash altogether since bash is required by `@system` anyway.

**Pull request closed without merging**